google/cloud/pubsub_v1/subscriber/client.py

```diff
@@ -123,11 +117,6 @@ def target(self) -> str:
         """
         return self._target
 
-    @property
-    def api(self) -> subscriber_client.SubscriberClient:
-        """The underlying gapic API client."""
-        return self._api
-
```

Comment:
Since this was documented

https://googleapis.dev/python/pubsub/latest/subscriber/api/client.html?highlight=api#google.cloud.pubsub_v1.subscriber.client.Client.api

we definitely need to consider this a breaking change. IMO it is worth taking that breaking change on, though.

Alternatively, you could update this to return `super()` or something.

Reply:
I agree, we should preserve this property.

I'm also thinking of deprecating it (docstring + a warning) now that the hand-written instances inherit all the essential stuff (including the gRPC channel, etc.).

Reply:
Adding it back with a deprecation sounds like the right plan to me